observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc

```diff
@@ -17,7 +17,13 @@ endif::[]
 
 include::modules/distr-tracing-tempo-config-default.adoc[leveloffset=+2]
 
-include::modules/distr-tracing-tempo-config-storage.adoc[leveloffset=+2]
+[role="_additional-resources"]
+[id="additional-resources_distr-tracing-tempo-config-default"]
+==== Additional resources
```

Comment:
```suggestion
.Additional resources
```
Use this format for additional resource following a module in an assembly, per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
Thank you so much, Ashleigh!
I'm giving it a try.